test/e2e/common/security_context.go

```diff
@@ -69,22 +69,21 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		}
 
 		/*
-		  Release : v1.12
-		  Testname: Security Context: runAsUser (id:65534)
-		  Description: Container created with runAsUser option, passing an id (id:65534) uses that
-		  given id when running the container.
-		  This test is marked LinuxOnly since Windows does not support running as UID / GID.
+			Release : v1.15
+			Testname: Security Context, runAsUser=65534
+			Description: Container is created with runAsUser option by passing uid 65534 to run as unpriviledged user. Pod MUST be in Succeeded phase.
+			[LinuxOnly]: This test is marked as LinuxOnly since Windows does not support running as UID / GID.
 		*/
-		It("should run the container with uid 65534 [LinuxOnly] [NodeConformance]", func() {
+		framework.ConformanceIt("should run the container with uid 65534 [LinuxOnly] [NodeConformance]", func() {
 			createAndWaitUserPod(65534)
 		})
 
 		/*
-		  Release : v1.12
-		  Testname: Security Context: runAsUser (id:0)
-		  Description: Container created with runAsUser option, passing an id (id:0) uses that
-		  given id when running the container.
-		  This test is marked LinuxOnly since Windows does not support running as UID / GID.
+			Release : v1.15
+			Testname: Security Context, runAsUser=0
+			Description: Container is created with runAsUser option by passing uid 0 to run as root priviledged user. Pod MUST be in Succeeded phase.
+			This e2e can not be promoted to Conformance because a Conformant platform may not allow to run containers with 'uid 0' or running privileged operations.
```

Comment:
@spiffxp @timothysc @johnbelamaric 

Have we started to tag tests with the reasons that tests aren't included in conformance yet,  such as `[Privileged]`? I thought we had agreed to do that:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md#kinds-of-tests